samples/joke/src/main/java/io/quarkiverse/operatorsdk/samples/joke/JokeReconciler.java

```diff
@@ -16,16 +16,76 @@
 
 import static io.javaoperatorsdk.operator.api.reconciler.Constants.WATCH_CURRENT_NAMESPACE;
 
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
 import io.javaoperatorsdk.operator.api.reconciler.Context;
 import io.javaoperatorsdk.operator.api.reconciler.ControllerConfiguration;
 import io.javaoperatorsdk.operator.api.reconciler.Reconciler;
 import io.javaoperatorsdk.operator.api.reconciler.UpdateControl;
+import io.quarkiverse.operatorsdk.samples.joke.JokeRequestSpec.ExcludedTopic;
+import io.quarkiverse.operatorsdk.samples.joke.JokeRequestStatus.State;
 
 @ControllerConfiguration(namespaces = WATCH_CURRENT_NAMESPACE)
-public class JokeReconciler implements Reconciler<Joke> {
+public class JokeReconciler implements Reconciler<JokeRequest> {
```

Comment:
It could make sense to rename this reconciler `JokeRequestReconciler` to make it clearer as you did before.

Reply:
Done